client/homescreen/activity-panel/index.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { Fragment } from '@wordpress/element';
+import { Accordion, AccordionPanel } from '@woocommerce/components';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { getUnreadOrders } from './orders/utils';
+import { getAllPanels } from './panels';
+
+export const ActivityPanel = () => {
+	const panels = useSelect( ( select ) => {
+		const countUnreadOrders = getUnreadOrders( select );
+		return getAllPanels( { countUnreadOrders } );
+	} );
+	return (
+		<Accordion>
+			<Fragment>
+				{ panels.map( ( panelData, index ) => {
+					const {
+						className,
+						count,
+						initialOpen,
+						panel,
+						title,
+					} = panelData;
+					return (
+						<AccordionPanel
+							key={ index }
```

Comment:
Not suggesting a change here but I just wanted to note that its *usually* a bad idea to use the index as the key. I think in this case its ok because we don't add/remove/reorder the list of panels, but its worth mentioning that if you were doing that you should have a stable unique ID for each item to use as the key. This video explains it well: https://egghead.io/lessons/react-use-the-key-prop-when-rendering-a-list-with-react

Reply:
Good idea! This was changed in the commit 0d10549